extensions/file/src/main/resources/META-INF/mule-module.properties

```diff
@@ -1,4 +0,0 @@
-module.name=file
-
-artifact.export.classPackages=org.mule.extension.file.api
-artifact.export.resourcePackages=/META-INF
```

Comment:
No longer needed, any test that needs to use this extension should be migrated to use isolation runner. So far, not even integration tests are using this API for their tests.